bittensor/_cli/commands/delegates.py

```diff
@@ -38,7 +38,7 @@ def show_delegates( delegates: List['bittensor.DelegateInfo'], width: Optional[i
             str(delegate.hotkey_ss58),
             str(delegate.take),
             str(delegate.owner_ss58),
-            str(delegate.nominators),
+            str(len(delegate.nominators)),
```

Comment:
what does `delegate.nominators` contain? a list of hotkeys that have also nominated the same delegator?

Reply:
Essentially they "nominate" the delegate